lib/ast-utils.js

```diff
@@ -24,6 +24,8 @@ const bindOrCallOrApplyPattern = /^(?:bind|call|apply)$/;
 const breakableTypePattern = /^(?:(?:Do)?While|For(?:In|Of)?|Switch)Statement$/;
 const thisTagPattern = /^[\s*]*@this/m;
 
+
+const COMMENTS_IGNORE_PATTERN = "^\\s*(?:eslint|jshint\\s+|jslint\\s+|istanbul\\s+|globals?\\s+|exported\\s+|jscs|falls?\\s?through)";
```

Comment:
I don't have a strong opinion on this either. My slight preference would be to change it to a regex for better readability (no need to double-escape things). Ideally, our documentation should be good enough that people don't need to look into the source code for examples.